eng/pipelines/templates/stages/archetype-ios-release.yml

```diff
@@ -0,0 +1,78 @@
+  # pipeline completes.
+  - ${{if and(eq(variables['Build.Reason'], 'Manual'), eq(variables['System.TeamProject'], 'internal'))}}:
+    - ${{ each artifact in parameters.Artifacts }}:
+      - stage: Release_${{artifact.safeName}}
+        displayName: 'Release: ${{artifact.name}}'
+        dependsOn: ${{parameters.DependsOn}}
+        condition: and(succeeded(), ne(variables['SetDevVersion'], 'true'), ne(variables['Skip.Release'], 'true'), ne(variables['Build.Repository.Name'], 'Azure/azure-sdk-for-java-pr'))
+        jobs:
+          - deployment: TagRepository
+            displayName: "Create release tag"
+            condition: and(succeeded(), ne(variables['Skip.TagRepository'], 'true'))
+            environment: github
+            variables:
+              - template: ../variables/globals.yml
+            pool:
+              name: Azure Pipelines
+              vmImage: vs2017-win2016
+            strategy:
+              runOnce:
+                deploy:
+                  steps:
+                    - checkout: self
+                    - template: /eng/common/pipelines/templates/steps/retain-run.yml
+                    # TODO: Will need to be adapted to work for Cocoapods (we'll read versions from the Cocoapod version specs)
+                    # - template: /eng/common/pipelines/templates/steps/create-tags-and-git-release.yml
+                    #   parameters:
+                    #     ArtifactLocation: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed/${{artifact.groupId}}/${{artifact.name}}
+                    #     PackageRepository: Maven
+                    #     ReleaseSha: $(Build.SourceVersion)
+
+          - deployment: PublishPackageToCocoaPodsTrunk
+            displayName: "Publish to CocoaPods Trunk"
+            condition: and(succeeded(), ne(variables['Skip.PublishPackage'], 'true'))
+            environment: cocoapodstrunk
+            dependsOn: TagRepository
+            variables:
+              - template: ../variables/globals.yml
+            pool:
+              name: Azure Pipelines
+              vmImage: $(OSVmImage)
+            strategy:
+              runOnce:
+                deploy:
+                  steps:
+                    - script: |
+                        echo Pretend we are publishing to CocoaPods truck.
+
+          - deployment: PublishPackageToSwiftPackageManagerMirrors
+            displayName: "Publish to CocoaPods Trunk"
+            condition: and(succeeded(), ne(variables['Skip.PublishPackage'], 'true'))
+            environment: cocoapodstrunk
+            dependsOn: TagRepository
+            variables:
+              - template: ../variables/globals.yml
+            pool:
+              name: Azure Pipelines
+              vmImage: $(OSVmImage)
+            strategy:
+              runOnce:
+                deploy:
+                  steps:
+                    - script: |
+                        echo Pretend we are publishing to Swift Package Manager mirrors
```

Comment:
The command used here is:
`python3 eng/scripts/sync_repo.py AzureFoo`

Followed by a git push. We should probably decide whether we want to mirror each push to master that affects a service directory, or just sync the repos on release.

Reply:
Of course, we also need to tag the SwiftPM-AzureFoo repo with the version and the `azure-sdk-for-ios` repo accordingly. I believe we are following the correct convention:
- SwiftPM-AzureFoo: tagged with `1.0.0-beta.x`
- azure-sdk-for-ios: tagged with `AzureFoo_1.0.0-beta.x`

Reply:
Will look at this in an upcoming PR.